src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -475,6 +497,7 @@ protected override ImmutableArray<PendingBranch> Scan(ref bool badRegion)
             CSharpCompilation compilation,
             Symbol symbol,
             BoundNode node,
+            Binder binder,
             Conversions conversions,
```

Comment:
>Conversions conversions, [](start = 12, length = 24)

do we still need to pass conversions here, or can we just grab them from the new `binder` parameter?

Reply:
I went back and forth on that and either approach seemed reasonable.

---
In reply to: [282702465](https://github.com/dotnet/roslyn/pull/35601#discussion_r282702465) [](ancestors = 282702465)